src/UI/Components/Diagram/helpers.ts

```diff
@@ -529,3 +531,71 @@ export const toggleLooseElement = (
     }),
   );
 };
+
+/**
+ * Gets the coordinates of all cells in the graph.
```

Comment:
Someone not familiar with jointJS might wonder what the graph is. Maybe specify that this is the canvas we are talking about?

Reply:
graph isn't exactly the canvas, should I add link to the documentation there?

Reply:
that would be nice